gradio/interface.py

```diff
@@ -769,6 +788,20 @@ def integrate(self, comet_ml=None, wandb=None, mlflow=None) -> None:
             utils.integration_analytics(data)
 
 
+class TabbedInterface(Blocks):
+    def __init__(
```

Comment:
Instead can the API be tuples of (interface: Interface, tabname: string)? Would prefer that over parallel arrays

Reply:
Ok I have no strong opinion, so I'll make the change

Reply:
Actually wait, it's more the way I have it because tab names can be optional

Reply:
Unless you have a strong reason for the other way, this seems more convenient for users.